### PR TITLE
nextinspace: init at 1.0.2

### PR DESCRIPTION
**Pull request by penguwin**
###### Motivation for this change

bring the [nextinspace](https://github.com/The-Kid-Gid/nextinspace) command-line tool to nixpkgs. This is my first contribution for a project written in python therefore I'm not really sure if this is packaged 'the right way'.

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/applications/science/misc/nextinspace/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+buildPythonPackage rec {
+  pname = "nextinspace";
+  version = "1.0.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0wrvdwpxwf9aq4mq377mmdlhbgy2ngzf58frp71jdbsipkhb2kcf";
+  };
+
+  pythonPath = with pythonPackages; [
```
[the remark this answers is not in the document]
Comment (penguwin) [reply pkgs/applications/science/misc/nextinspace/default.nix]: Thanks!

**Comment by penguwin** [comment]
@SuperSandro2000 thanks for the suggestions! I guess this can be reviewed again (I'm unable to re-request a review via the button)

**Comment by SuperSandro2000** [comment]
> @SuperSandro2000 thanks for the suggestions! I guess this can be reviewed again (I'm unable to re-request a review via the button)

You can't because I am not part of the github org.

**Comment by penguwin** [comment]
Closing as [v.1.0.3](https://github.com/The-Kid-Gid/nextinspace/releases/tag/v1.0.3) released.

**Pull request closed without merging**